docs/turn-howto.md

```diff
@@ -134,21 +150,35 @@ This will install and start a systemd service called `coturn`.
     traffic (remember to allow both TCP and UDP traffic), and ports 49152-65535
     for the UDP relay.)
 
-1.  We do not recommend running a TURN server behind NAT, and are not aware of
-    anyone doing so successfully.
+1.  If your TURN server is behind a NAT, the NAT gateway must have an external,
+    publicly-reachable IP address. You must configure coturn to advertise that
+    address to connecting clients:
+
+    ```
+    external-ip=EXTERNAL_NAT_IPv4_ADDRESS
+    ```
 
-    If you want to try it anyway, you will at least need to tell coturn its
-    external IP address:
+    You may optionally limit the TURN server to listen only on the local
+    address that is mapped by NAT to the external address:
 
     ```
-    external-ip=192.88.99.1
+    listening-ip=INTERNAL_TURNSERVER_IPv4_ADDRESS
+    external-ip=EXTERNAL_NAT_IPv4_ADDRESS/INTERNAL_TURNSERVER_IPv4_ADDRESS
```

Comment:
this `external/internal` notation is only required if there is more than one external IP address, right?

Reply:
That should be the case. I'll test what happens when using multiple `listening-ip`s with only one `external-ip`. If all public->private address pairings are made automatically, I'll remove line 166.

Reply:
After some testing, it looks like the config is much more flexible than I thought. The `external/internal` notation isn't needed with multiple active listening IPs, or even multiple relay IPs--as long as one paring works, just the `external` notation is enough.

I'll trim down this section accordingly.